utils/mil_msgs/msg/LabeledImage.msg

```diff
@@ -0,0 +1,7 @@
+# Represents the complete data from a labeled image from LabelMe
+
+sensor_msgs/Image image            # Image where objects are labeled
```

Comment:
Yeah I'm starting to agree with you... maybe this should be renamed LabeledObjects.msg and just contain a header, where the stamp is the same as when the image is produced and the frame_id is the frame_id of the camera. Than you just throw this in with the bag and the tools on the other ends sync them up.